Firestore/Protos/objc/firestore/local/MaybeDocument.pbobjc.h

```diff
@@ -1,19 +1,3 @@
-/*
```

Comment:
These edits that remove the copyright should be removed.

Reply:
https://guides.cocoapods.org/syntax/podspec.html#script_phases would be one way to integrate the nanopb build if it needs to be done outside of CocoaPods itself.

Reply:
Whoops. Fixed. (git amend'd and force pushed.)